testgrid/config.yaml

```diff
@@ -2516,13 +2516,27 @@ test_groups:
 dashboards:
 - name: conformance-all
   dashboard_tab:
-  - name: gci-gce
+  - name: gce
+    description: Runs conformance tests using kubetest against latest kubernetes master CI build on GCE
     test_group_name: ci-kubernetes-gce-conformance
+  - name: local-e2e
+    description: Runs conformance tests using kubetest with local-up-cluster
+    test_group_name: ci-kubernetes-local-e2e
 
 - name: conformance-gce
   dashboard_tab:
-  - name: gci-gce
+  - name: gce
+    description: Runs conformance tests using kubetest against latest kubernetes master CI build on GCE
     test_group_name: ci-kubernetes-gce-conformance
+    # TODO(bentheelder): there's might be a more appropriate alias to alert this to
+    alert_options:
+      alert_mail_to_addresses: 'gke-kubernetes-engprod+alerts@google.com'
```

Comment:
cc @mithrav @AishSundar: using this for alerting on GCP related conformance jobs for now